src/App.vue

```diff
@@ -50,7 +50,8 @@
     </v-fade-transition>
 
     <SbcHeader />
-    <PaySystemAlert />
+    <!-- TODO: revert before final commit -->
+    <PaySystemAlert v-if="false" />
```

Comment:
I'll go through all the TODOs in the next round (before final-final commit).

(This component generates a warning/error which was really annoying during development. Yes I created a ticket for this.)

Reply:
This is now reverted.